src/pages/dev/amplifier/chain-integration/introduction.mdx

```diff
@@ -7,57 +7,64 @@ The Axelar Virtual Machine (AVM) and Amplifier are currently under active develo
-
-- **Source Chain Gateway Contract**
-    - Built using the tech stack of your chain, you'll need to deploy a smart contract implementing the [Cross-Chain Gateway Protocol](https://github.com/axelarnetwork/cgp-spec).
-    - For EVM chains, you can re-use the existing [EVM Gateway](https://github.com/axelarnetwork/axelar-cgp-solidity).
-    - This contract serves as the primary API for developers sending messages from your chain.
-- **Verifier Contract**
-    - A smart contract on the Axelar network that verifies the validity of transactions on your chain. 
-    - Most integrators will begin by using or customizing a [Voting Verifier](https://github.com/axelarnetwork/axelar-amplifier/tree/main/contracts/voting-verifier) or can be customized to your needs for cases such as verification via ZK Proof.
-    - If you deploy a Voting Verifier, verifiers will need to support your chain and vote on the truth of source chain transactions.
-- **Gateway Contract**
-    - A gateway contract that exists on the Axelar network that will knows how to use your verifier contract to check the validity of a transaction and then knows how to forward transactions onto the Amplifier router.
-    - Most integrators will begin by using or customizing this [Gateway Contract](https://github.com/axelarnetwork/axelar-amplifier/tree/main/contracts/gateway).
-- **Prover Contract**
-    - A smart contract on the Axelar network that knows how to prove the validity of a transaction on your chain.
-    - Most integrators will begin by usign or customzing the [multisig prover](https://github.com/axelarnetwork/axelar-amplifier/tree/main/contracts/multisig-prover)
-- **Message Relayer**
-    - A service built with any tech stack that listens for events on your chain and relays them to the Axelar network.
-    - Get started with our [Message Relayer Example](https://github.com/axelarnetwork/axelar-examples/tree/main/examples/amplifier)
-
-
-The [chain integration tutorial](/dev/amplifier/chain-integration/integrate-a-chain) will guide you through this process within the current Amplifier DevNet. Contract deployment on Testnet or Mainnet will require governance proposal.
+
+To connect your chain to the Axelar network via the Interchain Amplifier, you’ll need to build or instantiate four smart contracts and one relayer service. The [chain integration tutorial](/dev/amplifier/chain-integration/integrate-a-chain) will guide you through this process on the current Amplifier devnet, which requires submitting a [whitelisting form](https://docs.google.com/forms/d/e/1FAIpQLSchD7P1WfdSCQfaZAoqX7DyqJOqYKxXle47yrueTbOgkKQDiQ/viewform) to the Interop Labs team at the end. Contract deployment on testnet or mainnet will require a governance proposal.
+
+### Source chain gateway contract
+
+- This is the gateway contract for your source chain, built using the same tech stack as that chain. It serves as the primary API for developers sending messages from your chain.
```

Comment:
"same tech stack as that chain" is not quite right. You use the tech stack of the chain. AKA Ethereum isn't built in Solidity, but the tech stack for smart contracts on Ethereum is Solidity.